frontend/src/components/EtherPrice.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
```

Comment:
(I'm guilty of not doing this on other components but...) For reusable components, PropTypes are handy for IDE code completion + sanity checking: https://reactjs.org/docs/typechecking-with-proptypes.html 

e.g.:`yarn add prop-types`

`import PropTypes from 'prop-types';`

```
EtherPrice. propTypes = {
    price: PropTypes.number.isRequired
}
```

Reply:
also unrelated and optional, but FYI in react there are things called stateless functional components that can make components that don't use state a little more readable: 
[here's a good primer](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc)